lib/ain-evm/src/storage/mod.rs

```diff
@@ -193,7 +193,17 @@ impl FlushableStorage for Storage {
 
 impl Storage {
     pub fn get_code_by_hash(&self, address: H160, hash: H256) -> Result<Option<Vec<u8>>> {
```

Comment:
Should we implement a CodeStorage trait since its used by both the cache and blockstore?